docs/sources/features/explore/index.md

```diff
@@ -51,6 +51,67 @@ In split view, timepickers for both panels can be linked (if you change one, the
 
 You can close the newly created query by clicking on the Close Split button.
 
+## Query history
+
+> BETA: Query history is a beta feature.
+
+Query history is a list of queries that you have used in Explore. The history is local to your browser and is not shared with others. To open and interact with your history, click the **Query history** button in Explore.
+
+### View query history
+
+Query history lets you view the history of your querying. For each individual query, you can:
+
+- Replace Explore query for selected query by clicking on the query card.
+- Create and/or edit a comment.
+- Copy a query to the clipboard.
+- Copy a URL link with the query to the clipboard.
+- Star a query.
+
+### Manage favorite queries
+
+All queries that have been starred in the Query history tab are displayed in the Starred. This allows you to access your favorite queries faster and to reuse these queries without typing them from scratch.
+
+### Sort query history
+
+By default, query history shows you the most recent queries. You can sort your history in by date or by data source name in ascending or descending order.
+
+1. Click the **Sort queries by** field.
+2. Select one of the following options:
+   - Time ascending
+   - Time descending
+   - Data source A-Z
+   - Data source Z-A
+
+> Note: If you are in split mode, then the chosen sorting mode applies only to the active panel.
+
+### Filter query history
+
+Filter query history in Query history and Starred tab by data source name:
+
+1. Click the **Filter queries for specific data source(s)** field
+2. Select the data source for which you would like to filter your history. You can select multiple data sources.
+
+In **Query history** tab is is also possible to filter queries by date using the slider:
+
+- Use vertical slider to filter queries by date.
+- By dragging top handle, adjust start date.
+- By dragging top handle, adjust end date.
+
+> Note: If you are in split mode, filters are applied only to your currently active panel.
+
+### Query history settings
+
+You can customize the query history in the Settings tab. Options are described in the table below.
+
+| Setting                                                       | Default value                           |
+| ------------------------------------------------------------- | --------------------------------------- |
+| Period of time for which Grafana will save your query history | 1 week                                  |
+| Change the default active tab                                 | Query history tab                       |
+| Only show queries for datasource currently active in Explore  | false                                   |
```

Comment:
Is `false` lowercase in the UI? Default values should match whatever is in the UI.

Reply:
there is a toggle switch - so I am not sure if it should be lower or upper cased 
![image](https://user-images.githubusercontent.com/30407135/76563431-afcd3a00-64a7-11ea-8958-3dad1b1e1052.png)